upgraderesponder/service.go

```diff
@@ -146,6 +160,13 @@ func (s *Server) initDB() error {
 	if err := s.createDB(InfluxDBDatabase); err != nil {
 		return err
 	}
+	// Uncomment this block if we support changing the InfluxDBContinuousQueryPeriod value by commandline argument
+	//if err := s.dropContinuousQueries(InfluxDBDatabase); err != nil {
```

Comment:
I'd rather have `continuous query` outside of code for now. We may have multiple pods running the same program at the same time.

Reply:
I want to put the `continuous query` into the code because we will have control over the measurements created by those `continuous queries`, e.g., `by_app_version_down_sampling`, `upgrade_request_down_sampling`. Knowing the measurements make it possible to build a Grafana dashboard template and user just have to import it.

Reply:
You meant we don't have control over original metrics? That's weird.

Reply:
Also, what does `uncomment this section` mean here? Does user have to recompile the code to do that?

Reply:
> You meant we don't have control over original metrics? That's weird.

Sorry for not being clear. 

The `continuous query` will periodically run on the original raw measurement `upgrade_request` and group data into 4 measurements `upgrade_request_down_sampling`, `by_app_version_down_sampling`, `by_kubernetes_version_down_sampling`, `by_country_code_down_sampling`. In our Grafana dashboard template, we query metrics from those 4 measurements. 

I meant that if we let the users to manually handle `continuous query`, they may pick different names for the 4 measurements and this will break the Grafana dashboard template. In this case, they have to log into Grafana and change the queries.



> Also, what does uncomment this section mean here? Does user have to recompile the code to do that? That's weird too.


Since InfluxDB doesn't allow us to change InfluxDBContinuousQueryPeriod of existing continuous queries, we have to drop the existing queries and create a new ones if we want to change InfluxDBContinuousQueryPeriod. 

Right now, we don't allow the user to change InfluxDBContinuousQueryPeriod value after it is set the first time. This comment means that if we want to allow the users to change InfluxDBContinuousQueryPeriod, we have to help them to drop the queries first, then recreate the queries. 

However, dropping queries is problematic as you mentioned that multiple pods will run this image at the same time. So, I removed the dropping logic entirely from the code.